bentoml/_internal/yatai_rest_api_client/schemas.py

```diff
@@ -4,6 +4,7 @@
 from typing import Dict
 from typing import List
 from typing import Type
+from typing import Literal
```

Comment:
This won't work for <3.7

Reply:
@aarnphm Do we support <3.7? But we use Literal a lot in our code.

https://github.com/bentoml/BentoML/blob/main/bentoml/grpc/types.py#L81

Reply:
Sorry i meant <3.8

Reply:
@aarnphm Understand, which are just annotation

Reply:
hence the block below so that we don't actually import `t.Literal` in runtime